src/slack/slack.service.ts

```diff
@@ -92,10 +92,19 @@ export class SlackService {
       code
     });
     if (response.ok && response.team?.id) {
+      const teamInfo = await this.webClient.team.info({ team: response.team.id });
```

Comment:
How is this API call working?
Have you set a value for `SLACK_BOT_TOKEN` in your .env?

Ideally we'll have to do something like the following

```suggestion
      const currentWorkspaceWebClient = new WebClient(response.access_token);
      const teamInfo = await currentWorkspaceWebClient.team.info({ team: response.team.id });

```

Reply:
The API call works since `this.webClient` was instantiated with the app‑level bot token (`SLACK_BOT_TOKEN`).
Yes, I have set the value for `SLACK_BOT_TOKEN` in my .env.

I used the existing bot‑level client for simplicity, but I can switch to a fresh `WebClient(response.access_token)` if you’d prefer each call to be scoped strictly to the installing workspace.

Reply:
>  but I can switch to a fresh WebClient(response.access_token) 

Let's do this

Reply:
Sure, I'll implement the change.